ftw/linkchecker/command/broken_link.py

```diff
@@ -28,12 +29,22 @@ def __init__(self):
         self.response_time = 'Unknown response time'
         self.error_message = 'No error occurred'
         self.creator = 'Unknown creator'
+        self.source_state = 'Unknown status'
 
     def __iter__(self):
         for attr in self.table_attrs:
             value = getattr(self, attr, '')
             yield value if isinstance(value, basestring) else ''
 
+    @staticmethod
+    def get_workflow_state(obj):
+        wftool = api.portal.get_tool('portal_workflow')
+        if ISimplelayoutBlock.providedBy(obj):
```

Comment:
Now every project using `ftw.linkchecker` must install `ftw.simplelayout`, which was only installed in the tests until now... 

I would remove the `if` and check the length of the workflow chain. If the length is zero, the object does not have a workflow assigned.

Reply:
That's true. I did not think about that. I'll change this to looking for the length.